rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -158,14 +159,15 @@ public static boolean isFreeBridgeTx(org.ethereum.core.Transaction rskTx, long b
                blockchainConfig.getConfigForBlock(blockNumber).areBridgeTxsFree() &&
                rskTx.acceptTransactionSignature() &&
                (
-                       isFromFederateMember(rskTx) ||
+                       isFromGenesisFederateMember(rskTx, bridgeConstants) ||
                        isFromFederationChangeAuthorizedSender(rskTx, bridgeConstants) ||
                        isFromLockWhitelistChangeAuthorizedSender(rskTx, bridgeConstants)
                );
     }
 
-    private static boolean isFromFederateMember(org.ethereum.core.Transaction rskTx) {
-        return true;
+    private static boolean isFromGenesisFederateMember(org.ethereum.core.Transaction rskTx, BridgeConstants bridgeConfiguration) {
```

Comment:
This sounds kind of odd to me. The method is called `isFromGenesisFederateMember` but needs to be fed the genesis federation from outside. So either one of two things:

1 - It doesn't actually have knowledge about the genesis federation, therefore the name is wrong. Remove the `Genesis` bit and pass the federation that is being checked for membership.
2 - Let the method gather the genesis federation itself.

I personally vote on option 1.

Cheers.

Reply:
I think I was following the other methods conventions, but I like your suggestion better. Just pushed and amended commit!

Reply:
Yeah, was just looking and both `isFromFederationChangeAuthorizedSender` and `isFromLockWhitelistChangeAuthorizedSender` probably need the same treatment. Future refactoring.